run.sh

```diff
@@ -14,16 +14,17 @@ then
     echo "Great, the simulator is already running!"
 else
     echo "Starting the simulator..."
-    $THIS_DIR/ros/src/styx/unity_simulator_launcher.sh &
+    cd $THIS_DIR/ros/src/styx
+    ./unity_simulator_launcher.sh &
```

Comment:
The script 'unity_siumulator_launcher' is doing something similar, why do you need to 'cd' before running it?

Reply:
previous code does not work on mac. it expects profile.tmp file to be in pwd.